ext/ClimaOceanSpeedyWeatherExt/speedy_weather_exchanger.jl

```diff
@@ -95,16 +92,33 @@ function interpolate_atmosphere_state!(interfaces, atmos::SpeedySimulation, coup
     Qse = exchanger.cpu_surface_state.Qs
     Qle = exchanger.cpu_surface_state.Qℓ
 
-    ConservativeRegridding.regrid!(ua,  ue,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(va,  ve,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Ta,  Te,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(qa,  qe,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(pa,  pe,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Qsa, Qse, regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Qla, Qle, regridder, exchange_areas)
+    ConservativeRegridding.regrid!(ue,  regridder, ua)
+    ConservativeRegridding.regrid!(ve,  regridder, va)
+    ConservativeRegridding.regrid!(Te,  regridder, Ta)
+    ConservativeRegridding.regrid!(qe,  regridder, qa)
+    ConservativeRegridding.regrid!(pe,  regridder, pa)
+    ConservativeRegridding.regrid!(Qse, regridder, Qsa)
+    ConservativeRegridding.regrid!(Qle, regridder, Qla)
     
     arch = architecture(exchange_grid)
     fill_exchange_fields!(arch, exchange_state, exchanger.cpu_surface_state)
 
     return nothing
-end
\ No newline at end of file
+end
+
+function compute_net_atmosphere_fluxes!(coupled_model::SpeedyCoupledModel)
+    atmos = coupled_model.atmosphere
+
+    # All the location of these fluxes will change
+    Qs = similarity_theory_fields.sensible_heat
+    Mv = similarity_theory_fields.water_vapor
+    Qu = total_fluxes.ocean.heat.upwelling_radiation 
+
+    regridder = coupled_model.interfaces.exchanger.regridder
+
+    ConservativeRegridding.regrid!(atmos.diagnostic_variables.physics.sensible_heat_flux,  transpose(regridder), vec(interior(Qs)))
+    ConservativeRegridding.regrid!(atmos.diagnostic_variables.physics.evaporative_flux,    transpose(regridder), vec(interior(Mv)))
+    ConservativeRegridding.regrid!(atmos.diagnostic_variables.physics.surface_longwave_up, transpose(regridder), vec(interior(Qu)))
```

Comment:
Should this really be regridded?

Reply:
I am not still not sure what we want to do with the upwelling longwave. We can either compute it in ClimaOcean or pass the temperature to the atmosphere which computes its upwelling radiation.
If we separate radiation it might be easier, but for the moment we can compute it in ClimaOcean and pass it to the atmosphere

Reply:
yeah, let's start with regridding also the longwave up which might be propedeutic for a radiation computed in the EarthSystemModel